.gitignore

```diff
@@ -205,6 +205,9 @@ node_modules/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
 
+
+/ajax/libs/angular-mocks/*-build*/
+
```

Comment:
The empty lines shouldn't be added, and this may not help conflict parts at all.
The correct method to avoid the conflicts is to add the meaningful line randomly in the set, not just at the end of the set.